src/languages/en.ts

```diff
@@ -800,6 +800,7 @@ export default {
             phrase3: 'and',
             phrase4: 'Privacy',
         },
+        returnToClassic: 'Return to Expensify Classic',
```

Comment:
```suggestion
        returnToClassic: 'Switch to Expensify Classic',
```

Discussed in Slack and decided to change the copy slightly.